scripts/deploy_scripts/instantiate.bash

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
+
+NEUTRON_RPC="${NEUTRON_RPC:-tcp://0.0.0.0:26657}"
+NEUTRON_HOME="${NEUTRON_HOME:-$HOME/.neutrond}"
+NEUTRON_CHAIN_ID="${NEUTRON_CHAIN_ID:-test-1}"
+TARGET_CHAIN_ID="${TARGET_CHAIN_ID:-test-2}"
+GAS_PRICES="${GAS_PRICES:-0.005}"
+KEYRING_BACKEND="${KEYRING_BACKEND:-test}"
+DEPLOY_WALLET="${DEPLOY_WALLET:-demowallet1}"
+MIN_NTRN_REQUIRED="${MIN_NTRN_REQUIRED:-10}"
+
+TARGET_SDK_VERSION="${TARGET_SDK_VERSION:?Variable should be explicitly specified}"
+TARGET_BASE_DENOM="${TARGET_BASE_DENOM:?Variable should be explicitly specified}"
+NEUTRON_SIDE_TRANSFER_CHANNEL_ID="${NEUTRON_SIDE_TRANSFER_CHANNEL_ID:?Variable should be explicitly specified}"
+INITIAL_VALIDATORS="${INITIAL_VALIDATORS:?Variable should be explicitly specified}"
+UNBONDING_PERIOD="${UNBONDING_PERIOD:?Variable should be explicitly specified}"
+UNBONDING_SAFE_PERIOD="${UNBONDING_SAFE_PERIOD:?Variable should be explicitly specified}"
+UNBOND_BATCH_SWITCH_TIME="${UNBOND_BATCH_SWITCH_TIME:?Variable should be explicitly specified}"
+PUPPETEER_TIMEOUT="${PUPPETEER_TIMEOUT:?Variable should be explicitly specified}"
+STAKER_TIMEOUT="${STAKER_TIMEOUT:?Variable should be explicitly specified}"
+
+NEUTRON_SIDE_PORT_ID="${NEUTRON_SIDE_PORT_ID:?Variable should explicitly specified}"
+NEUTRON_SIDE_UPDATE_PERIOD="${NEUTRON_SIDE_UPDATE_PERIOD:?Variable should explicitly specified}"
+SALT="${SALT:?Variable should explicitly specified}"
+SUBDENOM="${SUBDENOM:?Variable should explicitly specified}"
+TOKEN_METADATA_DESCRIPTION="${TOKEN_METADATA_DESCRIPTION:?Variable should explicitly specified}"
+TOKEN_METADATA_DISPLAY="${TOKEN_METADATA_DISPLAY:?Variable should explicitly specified}"
+TOKEN_METADATA_EXPONENT="${TOKEN_METADATA_EXPONENT:?Variable should explicitly specified}"
+TOKEN_METADATA_NAME="${TOKEN_METADATA_NAME:?Variable should explicitly specified}"
+TOKEN_METADATA_SYMBOL="${TOKEN_METADATA_SYMBOL:?Variable should explicitly specified}"
```

Comment:
These values can have default values, please use previous hard coded values as defaults for this variables